src/main/java/util/debugger/ValueJsonSerializer.java

```diff
@@ -188,12 +180,6 @@ private static boolean isSimpleValue(Value value) {
         }
     }
 
-    @NotNull
-    private static String toValRefString(ObjectReference valRef) {
```

Comment:
I tried it using that Tester's case I mentioned before, it resulted like this:
```
{
  ""{
  dvsltest.Tester@16b4a017=dvsltest.Tester@16b4a017,
  dvsltest.Tester@424c0bc4=dvsltest.Tester@3c679bde,
  dvsltest.Tester@3c679bde=dvsltest.Tester@424c0bc4
}"":{"name": "t3", "type": "hh", "obj": {"name": "t1", "type": "tt","obj": null}}, ""{dvsltest.Tester@16b4a017=dvsltest.Tester@16b4a017, dvsltest.Tester@424c0bc4=dvsltest.Tester@3c679bde, dvsltest.Tester@3c679bde=dvsltest.Tester@424c0bc4}"": {"name": "t2", "type": "pp", "obj": null},""{dvsltest.Tester@16b4a017=dvsltest.Tester@16b4a017, dvsltest.Tester@424c0bc4=dvsltest.Tester@3c679bde, dvsltest.Tester@3c679bde=dvsltest.Tester@424c0bc4}"":null
}
```
which is not a valid json string.

Reply:
fixed in [314eedd](https://github.com/chocovon/debug-variable-sl/pull/5/commits/314eedda6b063f01bd548d61decaa7f41a50977f)

now names are like in jackson library.

<img width="736" alt="Screenshot 2023-07-25 at 19 54 07" src="https://github.com/chocovon/debug-variable-sl/assets/644816/d0b1a0f6-dacb-46fe-bada-8179af6dd47a">